src/commands/contract/inspect_contract/mod.rs

```diff
@@ -0,0 +1,456 @@
+                    near_primitives::views::AccessKeyPermissionView::FullAccess
+                )
+            })
+            .count();
+        format!(
+            "{} full access keys and {} function-call-only access keys",
+            full_access_keys_count,
+            access_keys.len() - full_access_keys_count
+        )
+    };
+    table.add_row(prettytable::row![
+        Fy->"Access keys",
+        access_keys_summary
+    ]);
+
+    if let Ok(contract_source_metadata_response) = get_contract_source_metadata(
+        &network_config.network_name,
+        &json_rpc_client,
+        &block_reference,
+        account_id,
+    )
+    .await
+    {
+        if let Ok(contract_source_metadata) = contract_source_metadata_response
+            .call_result()?
+            .parse_result_from_json::<self::contract_metadata::ContractSourceMetadata>(
+        ) {
+            table.add_row(prettytable::row![
+                Fy->"Contract version",
+                contract_source_metadata.version.unwrap_or_default()
+            ]);
+            table.add_row(prettytable::row![
+                Fy->"Contract link",
+                contract_source_metadata.link.unwrap_or_default()
+            ]);
+            table.add_row(prettytable::row![
+                Fy->"Supported standards",
+                contract_source_metadata.standards
+                    .iter()
+                    .fold(String::new(), |mut output, standard| {
+                        let _ = writeln!(output, "{} ({})", standard.standard, standard.version);
+                        output
+                    })
+            ]);
+        }
+
+        if let Ok(contract_abi_response) = get_contract_abi(
+            &network_config.network_name,
+            &json_rpc_client,
+            &block_reference,
+            account_id,
+        )
+        .await
+        {
+            let call_result = contract_abi_response.call_result()?;
+            if let Ok(abi_root) = serde_json::from_slice::<near_abi::AbiRoot>(&zstd::decode_all(
+                &call_result.result[..],
+            )?) {
+                table.add_row(prettytable::row![
+                    Fy->"Schema version",
+                    abi_root.schema_version
+                ]);
+                table.add_row(prettytable::row![Fy->"Functions:"]);
+                table.printstd();
+
+                for function in abi_root.body.functions {
+                    let mut table_func = prettytable::Table::new();
+                    table_func.set_format(*prettytable::format::consts::FORMAT_CLEAN);
+                    table_func.add_empty_row();
+
+                    if let near_abi::AbiParameters::Borsh { args: _ } = function.params {
+                        panic!("Borsh is currently unsupported")
+                    }
+                    table_func.add_row(prettytable::row![format!(
+                        "{} (read-write function - {}) {}\n{}",
```

Comment:
It must be `read-only function` for "view" functions

Reply:
Done